server/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoModel = require('../database/index');
+const app = express();
+
+
+
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+//REMEMBER TO ADD STATIC PATH
+
+//REMEMBER TO FIX FIND ONE BELOW!!!!!!!!!!!!
+app.get('/videoData', (req, res) => {
+  mongoModel.GameModel.findOne({id: 1})
```

Comment:
With url params you can just use restructured object notation

var id = req.params.id;
{ id }